### PR TITLE
added "node start" and updated readme with install & run instructions

### PR DESCRIPTION
**Pull request by michaelguild13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,12 @@
...
 Written for 24ways. [Go read the blog post](https://24ways.org/2015/universal-react/) :)
+
+## Install
+git clone https://github.com/jackfranklin/universal-react-example.git
+
+npm install
+
+npm start
+
+goto: localhost:3003
```
Comment (jackfranklin) [inline README.md]: do you fancy just putting these in backticks for nicer formatting?

```
git clone
npm install
npm start
open http://localhost:3003
```

?

**Comment by jackfranklin** [comment]
One comment but otherwise looks great, thank you so much!

File: README.md (hunk 1 above)
Comment (michaelguild13) [inline README.md]: Sure thing :)

**Comment by jackfranklin** [comment]
:+1:

**Pull request merged**